src/components/issue-list-item.component.js

```diff
@@ -99,10 +103,10 @@ export const IssueListItem = ({ type, issue, navigation, locale }: Props) => (
         titleStyle={styles.title}
         subtitleStyle={styles.subtitle}
       />
```

Comment:
Once #619 is merged, this `<ListItem>` can be converted as a styled component using this:

```js
const Issue = styled(ListItem).attrs({
  containerStyle: {
     // old style attributes
  },
  titleStyle: {
     // old style attributes
  },
  subtitleStyle: {
     // old style attributes
  }
})``;
```

Reply:
#619 is merged, you should now be able to convert the react-native-elements components!